acceptance-tests/main.test.ts

```diff
@@ -0,0 +1,130 @@
+import {describe,it,before,after} from 'mocha';
+import expect from 'expect';
```

Comment:
Again, additional dependency that is already introduced by jest.

Reply:
We wanted to use the jest matchers, despite using mocha.